### PR TITLE
Feature/1 add files

### PR DESCRIPTION
**Pull request by hhedden-isovera**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+
+## Prerequisites
+
+You will need the following to execute this script:
+
+- GitHub user account
+- GitHub personal access token, if you have two factor authentication enabled.
+  See [Creating a personal access token for the command line](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
+- Clone this repository or copy `enlabeler.sh`  to your local environment.
+- Install [jq](https://stedolan.github.io/jq/) with the command `brew install jq`.
```
Comment (amanire) [inline README.md]:
```suggestion
- Install [jq](https://stedolan.github.io/jq/) (on MacOS with [Homebrew](https://brew.sh/), use the command `brew install jq`).
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
+
+## To Use
+
+1. GitHub issue labels are assumed to use Code > Feature QA > Code Review convention as documented in
```
Comment (amanire) [inline README.md]: @hhedden-isovera is this line relevant without the ZenHub work? I don't think it is.

File: enlabeler.sh [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+	# Get the GitHub repository ID
+	repo_ID=$(curl https://api.github.com/repos/"$REPO_USER"/"$REPO_NAME" | jsonValue id)
+	repo_ID=${repo_ID:0:10} # Grab the repo ID number
+
+	# Get range of issue nums
+	issue_nums=$(curl https://api.github.com/repos/"$REPO_USER"/"$REPO_NAME/issues" | jsonValue number)
+	max_issue=${issue_nums:0:2} # Grab the latest (i.e highest) active issue number
+
+	#Grab the pipeline IDs
```
Comment (amanire) [inline enlabeler.sh:83]: This work looks like it can also be removed.

**Review by amanire — CHANGES_REQUESTED** [review]
a little more cleanup before merging

File: README.md (hunk 2 above)
Comment (hhedden-isovera) [reply README.md]: I'm pretty sure it's no longer relevant.

**Review by amanire — APPROVED** [review]

**Pull request merged**